### PR TITLE
Update go version in go.mod to 1.13

### PR DESCRIPTION
**Pull request by odinuge**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

We now use go 1.13 and not 1.12

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/assign liggitt

**Does this PR introduce a user-facing change?**:

```release-note
NONE

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 module k8s.io/kubernetes
 
-go 1.12
+go 1.13
```
Comment (mattjmcnaughton) [inline go.mod]: The `build/root/WORKSPACE` configuration specifies exactly `go_version=1.13.4`. Worth doing the same here?

**Review by mattjmcnaughton — COMMENTED** [review]
I _think_ this looks good to me! If I'm reading https://github.com/kubernetes/kubernetes/pull/82809 correctly, the rest of Kubernetes is already using 1.13.

@liggitt do you mind sharing more on whether now is the appropriate time to update `go.mod`?

/assign @liggitt 

/test pull-kubernetes-e2e-gce

File: go.mod (hunk 1 above)
Comment (odinuge) [reply go.mod]: Only major+minor is used, so when running go 1.13.4, you get the following result:
```bash
> go mod init example
go: creating new go.mod: module example
> cat go.mod  
module example

go 1.13
```

So we should stick to `go 1.13`

**Comment by odinuge** [comment]
/retest

Flaky flake day :man_shrugging:

**Comment by liggitt** [comment]
Can you make sure there's an open issue for the flaking test and tag the appropriate sig?

**Comment by odinuge** [comment]
> Can you make sure there's an open issue for the flaking test and tag the appropriate sig?

Currently looking into it. Have a slight feeling that https://github.com/kubernetes/test-infra/pull/15516 may have made the flakes more visible now that we only run them once, but it will hopefully help us make the test suites even more stable.

**Comment by liggitt** [comment]
I'm sure that is why we're seeing more flakes. Tracking open issues for specific flaky tests is the next step to driving our CI jobs to be more reliable.

**Pull request merged**